ha_client.py

```diff
@@ -196,6 +196,34 @@ def find_entity_attr(self, entity):
                     return entity_attr
         return None
 
+    def list_entities(self, types):
+        """List all entities matching domains used within our skill
+        and return them as list.
+
+        Throws request Exceptions
+        (Subclasses of ConnectionError or RequestException,
+          raises HTTPErrors if non-Ok status code)
```

Comment:
It doesn't look like we catch these anywhere?

I presume this would occur if the HA server is unavailable?

Should we catch and return None? 
Catch and speak an error dialog?

Reply:
There is '_handle_client_exception()' function but I can't say if it works or not or how. :smile: